netty/src/main/java/io/grpc/netty/CancelClientStreamCommand.java

```diff
@@ -39,17 +39,17 @@
  * Command sent from a Netty client stream to the handler to cancel the stream.
  */
 class CancelClientStreamCommand extends WriteQueue.AbstractQueuedCommand {
-  private final NettyClientStream stream;
+  private final NettyClientStream.TransportState stream;
```

Comment:
I'm not sure we want to. `Stream.TransportState` _is_ the stream from the transport's perspective, so `stream` isn't really _wrong_ in my mind (it is a drop-in replacement for `AbstractStream` in the transport, after all). `state` is just bad, and `transportState` seemed verbose given how often it is used and is still problematic since `transportState` could describe _anything_ in the transport. `streamTransportState` is verbose, but has no clarity problems.

I'm fine with changing it to _something_, but the name is used _a lot of places_ (including server-side, which is using `stream` today) so I want to make sure the new name works well.